catalog.html

```diff
@@ -0,0 +1,322 @@
+              <label for="service3">
+                <input id="service3" type="checkbox" name="Wi-Fi">Wi-Fi
+              </label>
+            </li>
+          </ul>
+        </div>
+
+        <div class="choice2">
+          <h2 class="catalog-filter-title">Тип жилья:</h2>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <label for="place1">
+                <input id="place1" type="radio" name="Гостиница" value="shaving-supplies" checked>Гостиница
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="place2">
+                <input id="place2" type="radio" name="Мотель" value="shaving-supplies">Мотель
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="place3">
+                <input id="place3" type="radio" name="Апартаменты" value="shaving-supplies">Апартаменты
+              </label>
+            </li>
+          </ul>
+        </div>
+
+        <div class="choice3">
+          <h2 class="catalog-filter-title">Стоимость, ₽:</h2>
+          <label for="format1">
+            <input id="format1" type="text" placeholder="от">
+          </label>
+          <label for="format2">
+            <input id="format2" type="text" placeholder="до">
+          </label>
+
+          <label for="roller">
+            <input type="range" id="roller" name="cost" min="0" max="9000" step="1000">
+          </label>
+        </div>
+
+        <button class="button" type="submit">Применить</button>
+        <label for="discard">
+          <input id="discard" type="reset" value="Сбросить">
+        </label>
+      </form>
+    </section>
+
+    <form action="https://echo.htmlacademy.ru/" method="get"></form>
+    <section>
+
+      <span>Найдено гостиниц:</span><output name="number">38</output>
+
+      <select name="select-control" aria-label="Сортировка">
+        <option value="popular">Сначала популярное</option>
+        <option value="cheap">Сначала дешёвые</option>
+        <option value="expensive">Сначала дорогие</option>
+      </select>
+
+      <a href="#" class="button button-frame"><img src="" alt="Показать список."></a>
+      <a href="#" class="button"><img src="" alt="Показать слайды."></a>
+      <a class="button" href="#"><img src="" alt="Показать карточки."></a>
+
+      <div class="cards">
+        <ul class="housing-list">
+          <li class="housing-cards">
+            <img src="" alt="Гостиница в скале.">
+            <a class="housing-cards-link" href="housing.html">
+              <h3 class="housing-cards-title">Amara Resort&Spa</h3>
+            </a>
+            <a href="#" class="housing">Гостиница</a>
+            <a href="#" class="control">от 4000 ₽</a>
+
+            <form action="https://echo.htmlacademy.ru/" method="post"></form>
+            <button type="button" formtarget="housing-cards">Подробнее</button>
```

Comment:
```suggestion
            <button type="button">Подробнее</button>
```

это тут лишний атрибут. И подумай ещё в такую сторону: что будет с пользователем, когда он кликнет по Подробнее? Он перейдёт на другую страницу? На этой странице будет более детальное описание гостиницы, например. И если мы переходим на другую страницу - это что? Ссылка?

Reply:
Да у меня сначала была ссылка, но потом я решила что когда происходит клик должно произойти расширение окошка с указанными дополнениями, потому сделала кнопку.

Reply:
Да, логика правильная, если бы  блочек с гостиницей открывался вниз, то надо было бы делать кнопку, но тут как в магазине, не знаю, пользовалась ли ты Яндекс.Маркетом или чем-то подобным - там обычно это переход на другую страницу. Поэтому тут ссылка